src/middlewared/middlewared/utils/rootfs.py

```diff
@@ -32,12 +32,13 @@ def readonly_source(self):
 
 
 class ReadonlyRootfsManager:
-    def __init__(self, root="/"):
+    def __init__(self, root="/", force_rw=False):
```

Comment:
Please document what does this option do. Also, maybe we should call it `force_usr_rw`?